### PR TITLE
arm-trusted-firmware: set DEBUG and LOG_LEVEL from bitbake

### PR DESCRIPTION
**Pull request by atharvanan1**
- this allows ability to modify these in other layers if needed

### CODE REVIEW

File: recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_2.8-l4t-r36.3.0.bb [hunk 1]
```diff
@@ -53,8 +53,10 @@ def generate_build_timestamp(d):
 BUILD_STRING ?= "${@generate_build_string(d)}"
 BUILDTIMESTAMP ?= "${@generate_build_timestamp(d)}"
 
+DEBUG ?= "0"
```
Comment (ichergui) [inline recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_2.8-l4t-r36.3.0.bb:56]: I prefer to have those variables specific to `ATF`.
It should be something as below:
```
ATF_DEBUG ?= "0"
ATF_LOG_LEVEL ?= "20"
```

**Review by ichergui — CHANGES_REQUESTED** [review]

**Comment by atharvanan1** [comment]
I'll move this to `scarthgap`, `scarthgap-l4t-35.x` and `kirkstone`

**Comment by atharvanan1** [comment]
@ichergui Please merge this whenever it seems fit.

**Comment by ichergui** [comment]
@atharvanan1 I don't have the permissions to merge this PR. I'm waiting for @madisongh to do that

**Comment by atharvanan1** [comment]
Thanks for review and merge on this! @ichergui and @madisongh

**Pull request merged**